src/server/components/dashboard/listsItems/item/update/actionHandlers/requestChanges.ts

```diff
@@ -0,0 +1,108 @@
+import { ListItemGetObject, User } from "server/models/types";
+import { logger } from "server/services/logger";
+import { initialiseFormRunnerSession } from "server/components/formRunner/helpers";
+import { getListItemContactInformation } from "server/models/listItem/providers/helpers";
+import serviceName from "server/utils/service-name";
+import { sendEditDetailsEmail } from "server/services/govuk-notify";
+import { AuditEvent, ListItem, Status } from "@prisma/client";
+import { prisma } from "server/models/db/prisma-client";
+import { EVENTS } from "server/models/listItem/listItemEvent";
+import { recordListItemEvent } from "server/models/audit";
+import { ListItemRes } from "server/components/dashboard/listsItems/types";
+import { Request } from "express";
+
+/**
+ * TODO: remove underTest
+ */
+export async function requestChanges(req: Request, res: ListItemRes) {
+  const { underTest } = req.params;
+  const isUnderTest = underTest === "true";
+  const userId = req.user!.id;
+  const changeMessage = req.session.update?.message;
+  const { listItem } = res.locals;
+  const { listItemUrl, listIndexUrl } = res.locals;
+
+  if (!listItem) {
+    logger.error("List item could not be found during requestChanges");
+    return res.status(500);
+  }
+
+  if (!changeMessage) {
+    req.flash("errorMsg", "You must provide a message to request a change");
+    return res.redirect(listItemUrl);
+  }
+
+  const jsonData = listItem.jsonData as ListItemGetObject["jsonData"];
+
+  try {
+    await handleListItemRequestChanges(listItem, changeMessage, userId, isUnderTest);
+
+    req.flash("successBannerTitle", `Change request sent to ${jsonData?.organisationName}`);
+    req.flash("successBannerHeading", "Requested");
+    req.flash("successBannerColour", "blue");
+    return res.redirect(listIndexUrl);
+  } catch (error: any) {
+    req.flash("errorMsg", `${jsonData?.organisationName} could not be updated. ${error.message}`);
+    return res.redirect(listItemUrl);
+  }
+}
+
+async function handleListItemRequestChanges(
```

Comment:
this could (and probably should) be renamed but it's just a lift and shift for now